ISL.Providers.Storages.AzureBlobStorage.Tests.Unit/Providers/AzureBlobStorage/AzureBlobStorageTestsExceptions.ListFilesInContainer.cs

```diff
@@ -0,0 +1,177 @@
+using Moq;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Xeptions;
+
+namespace ISL.Providers.Storages.AzureBlobStorage.Tests.Unit.Providers.AzureBlobStorage
+{
+    public partial class AzureBlobStorageTests
+    {
+        [Fact]
+        public async Task ShouldThrowProviderValidationExceptionOnListFilesInContainer()
+        {
+            // given
+            string randomContainer = GetRandomString();
+            string inputContainer = randomContainer;
+
+            var storageValidationException = new StorageValidationException(
+                message: "Storage validation error occurred, please fix errors and try again.",
+                innerException: new Xeption());
+
+            var expectedAzureBlobStorageProviderValidationException =
+                new AzureBlobStorageProviderValidationException(
+                    message: "Azure blob storage provider validation error occurred, fix errors and try again.",
+                    innerException: (Xeption)storageValidationException.InnerException,
+                    data: storageValidationException.InnerException.Data);
+
+            this.storageServiceMock.Setup(service =>
+                service.ListFilesInContainerAsync(inputContainer))
+                    .ThrowsAsync(storageValidationException);
+
+            // when
+            ValueTask<List<string>> listFilesTask =
+                this.azureBlobStorageProvider.ListFilesInContainerAsync(inputContainer);
+
+            AzureBlobStorageProviderValidationException actualAzureBlobStorageProviderValidationException =
+                await Assert.ThrowsAsync<AzureBlobStorageProviderValidationException>(
+                    testCode: listFilesTask.AsTask);
+
+            // then
+            actualAzureBlobStorageProviderValidationException
+                .Should().BeEquivalentTo(expectedAzureBlobStorageProviderValidationException);
+
+            this.storageServiceMock.Verify(service =>
+                service.ListFilesInContainerAsync(inputContainer),
+                    Times.Once);
+
+            this.storageServiceMock.VerifyNoOtherCalls();
+        }
+
+        [Fact]
+        public async Task ShouldThrowProviderDependencyValidationExceptionOnListFilesInContainer()
+        {
+            // given
+            string randomContainer = GetRandomString();
+            string inputContainer = randomContainer;
+
+            var storageDependencyValidationException = new StorageDependencyValidationException(
+                message: "Storage dependency validation error occurred, please fix errors and try again.",
+                innerException: new Xeption());
+
+            var expectedAzureBlobStorageProviderDependencyValidationException =
+                new AzureBlobStorageProviderDependencyValidationException(
+                    message: "Azure blob storage provider dependency validation error occurred, " +
+                            "fix errors and try again.",
```

Comment:
fix indentation:

```suggestion
                    message: "Azure blob storage provider dependency validation error occurred, " +
                        "fix errors and try again.",
```

OR

```suggestion
                    message: 
                        "Azure blob storage provider dependency validation error occurred, " +
                        "fix errors and try again.",
```

Reply:
this has been updated